github-event-service/src/push.ts

```diff
@@ -0,0 +1,56 @@
+import * as fetch from 'node-fetch';
+import {safeLoad} from 'js-yaml';
+
+import {getFirebaseInst} from './firebase-init';
+
+/** The settings object for the branch manager. */
+export interface BranchManagerSettings {
+  enabled: boolean;
+  branches: Array<{
+    branch: string;
+    label: string;
+  }>;
+};
+
+/** Limited implementation of the Github push webhook event. */
+export interface GithubPushEvent { 
+  ref: string;
+  repository: {
+    id: number;
+    full_name: string;
+  }
+};
+
+/** The Firebase app. */
+const firestore = getFirebaseInst().firestore();
+
+/** Handle webhook events from github of the type, `push`. */
+export const handlePushEvent = (event: GithubPushEvent) => {
```

Comment:
Why are these constants and not just real `functions`? It doesn't make a big difference, just out of curiosity. Same for other methods and the other PR.

Reply:
Just out of habit I guess, switched it to normal `function` declarations